IdentityCore/tests/MSIDAssymetricKeychainGeneratorTests.m

```diff
@@ -135,9 +125,6 @@ - (void)testGenerateKeyPair_whenKeyExists_shouldGenerateNewKeyAndReturnIt
     
     BOOL privateKeyExists = [self keyExists:privateKeyIdentifier];
     XCTAssertTrue(privateKeyExists);
-    
-    BOOL publicKeyExists = [self keyExists:publicKeyIdentifier];
```

Comment:
Should we instead check that we were able to retrieve public key using our APIs?